src/config/transform.rs

```diff
@@ -236,3 +237,23 @@ pub trait TransformConfig: DynClone + NamedComponent + core::fmt::Debug + Send +
 }
 
 dyn_clone::clone_trait_object!(TransformConfig);
+
+/// Often we want to call outputs just to retreive the OutputId's without needing
+/// the schema definitions.
+pub fn transform_output_ids<T: TransformConfig + ?Sized>(
```

Comment:
non-blocking: the function name could be interpreted multiple ways (it transforms output IDs vs retrieves output IDs from transforms).
This could be avoided by a rename to something like `get_transform_output_ids()` or `output_ids_of_transform()`